### PR TITLE
Puts DP on Bitmasks and Ranges in Gold 

### PR DESCRIPTION
**Pull request by Aryansh-S**
Moves from plat to gold and updates ordering.ts as discussed in the issue earlier

### CODE REVIEW

**Review by bqi343 — COMMENTED** [review]
might want to rename DP on Bitmasks / DP on Ranges to Bitmask DP and Range DP respectively to be consistent with "Knapsack DP"?

**Comment by Aryansh-S** [comment]
> might want to rename DP on Bitmasks / DP on Ranges to Bitmask DP and Range DP respectively to be consistent with "Knapsack DP"?

Done, go ahead and check if this works and merge if everything checks out

**Review by bqi343 — APPROVED** [review]
will wait for some more reviews

**Review by dongliu0426 — APPROVED** [review]

**Review by dolphingarlic — APPROVED** [review]

**Comment by dolphingarlic** [comment]
Should I merge?

**Comment by thecodingwizard** [comment]
Can we add redirects from the old URL? Front matter accepts a redirects field that's an array of old URL paths

**Comment by Aryansh-S** [comment]
> Can we add redirects from the old URL? Front matter accepts a redirects field that's an array of old URL paths
> 
> Also does frequency need to be updated?

Probably a good idea to mention the people who handle that

File: content/4_Gold/DP_Bitmasks.mdx
Comment (thecodingwizard) [inline content/4_Gold/DP_Bitmasks.mdx:6]: Should add a redirect somewhere in the frontmatter:

```
redirects:
  - /plat/dp-bitmasks
```

**Review by thecodingwizard — CHANGES_REQUESTED** [review]

**Comment by thecodingwizard** [comment]
@Aryansh-S I think we can just set frequency to 0 for now unless somebody else has a different opinion?

**Review by Farrius — APPROVED** [review]

**Comment by andrewwangva** [comment]
> @Aryansh-S I think we can just set frequency to 1 for now (for both modules) unless somebody else has a different opinion?

1 seems good

**Comment by Aryansh-S** [comment]
I've changed the frequencies for both to 1. @thecodingwizard after your review, I think you should merge

**Pull request merged**
